app/src/main/java/com/gusta/wakemehome/IAlarmAdapterListeners.java

```diff
@@ -0,0 +1,8 @@
+package com.gusta.wakemehome;
+
+import com.gusta.wakemehome.database.AlarmEntry;
+
+public interface IAlarmAdapterListeners {
+    void onItemClickListener(int itemId);
+    void onAlarmEnabledChangeListener(AlarmEntry alarm);
+}
```

Comment:
Why did you move it to a new file? This interface can't live by it's own - it's an internal interface the adapter class is publishing for other components so they can use the adapter.